specs/git/remove_local_branches_not_in_remote.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Remove all local branches that aren't in remote
```

Comment:
maybe "Remove all local branches that aren't on the remote repository"?

Reply:
Ok, updated.